mobsf/StaticAnalyzer/views/android/converter.py

```diff
@@ -90,8 +92,11 @@ def apk_2_java(app_path, app_dir, tools_dir):
             app_path,
         ]
         fnull = open(os.devnull, 'w')
-        subprocess.call(args,
-                        stdout=fnull,
-                        stderr=subprocess.STDOUT)
+        subprocess.run(args,
+                       stdout=fnull,
+                       stderr=subprocess.STDOUT,
+                       timeout=get_jadx_timeout_value())
```

Comment:
Today we read all our values from `settings.py` and not use environment variable for settings.
We do plan to do that with a later PR to address this (https://github.com/MobSF/Mobile-Security-Framework-MobSF/issues/1889)

Can this be moved to `settings.py` with a sensible default like 1800 (30 minutes).  The function `get_jadx_timeout_value` can be avoided here.

Reply:
Nice, thanks, edited.